### PR TITLE
Framework is working correctly

### PR DESCRIPTION
**Pull request by benjamin-confino**

### CODE REVIEW

**Comment by benjamin-confino** [comment]
#build

File: dev/com.ibm.ws.microprofile.reactive.messaging_git_fat_tck/bnd.bnd [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	fat/src
+
+fat.project: true
+
+// This list does not need to be exhaustive, here we cover the fact that
+// we have   <feature>mpFaultTolerance-${server.xml.feature.version}</feature>
+// in server.xml and also include some currently non existing levels.
+tested.features: concurrent-1.0,\
+                 mpFaultTolerance-1.0,\
+                 mpFaultTolerance-1.1
```
Comment (Azquelt) [inline dev/com.ibm.ws.microprofile.reactive.messaging_git_fat_tck/bnd.bnd]: This list looks wrong

File: dev/com.ibm.ws.microprofile.reactive.messaging_git_fat_tck/fat/src/com/ibm/ws/microprofile/faulttolerance/tck/ReactiveStreamsMessageingGitTckLauncher.java [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+     * Various TCK tests test for Deployment, Definition and other Exceptions and
+     * these will cause the test suite to be marked as FAILED if found in the logs
+     * when the server is shut down. So we tell Simplicity to allow for the message
+     * ID's below.
+     *
+     * @throws Exception
+     */
+    @AfterClass
+    public static void tearDown() throws Exception {
+        server.stopServer("CWMFT5001E", // CWMFT0001E: No free capacity is available in the bulkhead
```
Comment (Azquelt) [inline dev/com.ibm.ws.microprofile.reactive.messaging_git_fat_tck/fat/src/com/ibm/ws/microprofile/faulttolerance/tck/ReactiveStreamsMessageingGitTckLauncher.java]: These messages look like they refer to a different TCK

File: dev/com.ibm.ws.microprofile.reactive.messaging_git_fat_tck/fat/src/com/ibm/ws/microprofile/faulttolerance/tck/ReactiveStreamsMessageingGitTckLauncher.java [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+        addedProps.put(MvnUtils.IMPL_VERSION, implVersion);
+
+        // We store a set of keys that we want the system to add "1.1" or "1.2" etc to
+        // depending on the pom.xml contents.
+        HashSet<String> versionedLibraries = new HashSet<>(Arrays.asList("org.eclipse.microprofile.reactive.messaging"));
+        String backStopImpl = "1.0"; // Used if there is no impl matching the spec/pom.xml <version> AND impl.version is not set
+        addedProps.put(MvnUtils.BACKSTOP_VERSION, backStopImpl);
+
+        MvnUtils.runTCKMvnCmd(server, "org.eclipse.microprofile.reactive.messaging.tck", this
+                        .getClass() + ":launchFaultToleranceTCK", MvnUtils.DEFAULT_SUITE_FILENAME, addedProps, versionedLibraries);
```
Comment (Azquelt) [inline dev/com.ibm.ws.microprofile.reactive.messaging_git_fat_tck/fat/src/com/ibm/ws/microprofile/faulttolerance/tck/ReactiveStreamsMessageingGitTckLauncher.java]: Not `launchFaultToleranceTCK`

File: dev/com.ibm.ws.microprofile.reactive.messaging_git_fat_tck/publish/tckRunner/pom.xml [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.ibm.ws.microprofile.faulttolerance</groupId>
+    <artifactId>tck.runner</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>pom</packaging>
+    <name>MicroProfile Fault Tolerance TCK Runner</name>
```
Comment (Azquelt) [inline dev/com.ibm.ws.microprofile.reactive.messaging_git_fat_tck/publish/tckRunner/pom.xml]: Not Fault Tolerance

File: dev/com.ibm.ws.microprofile.reactive.messaging_git_fat_tck/publish/tckRunner/pom.xml [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.ibm.ws.microprofile.faulttolerance</groupId>
```
Comment (Azquelt) [inline dev/com.ibm.ws.microprofile.reactive.messaging_git_fat_tck/publish/tckRunner/pom.xml]: Update groupId

File: dev/com.ibm.ws.microprofile.reactive.messaging_git_fat_tck/publish/tckRunner/tck/pom.xml [hunk 6]
```diff
@@ -0,0 +1,201 @@
...
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.ibm.ws.microprofile.reactive.streams</groupId>
+    <artifactId>tck.runner.tck</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <name>MicroProfile Reactive Streams TCK Runner TCK Module</name>
+
+    <properties>
+        <microprofile.reactive.streams.version>1.0</microprofile.reactive.streams.version>
+        <microprofile.reactive.messageing.version>1.0-SNAPSHOT</microprofile.reactive.messageing.version>
```
Comment (Azquelt) [inline dev/com.ibm.ws.microprofile.reactive.messaging_git_fat_tck/publish/tckRunner/tck/pom.xml]: messageing -> messaging - also wherever it's referenced

File: dev/com.ibm.ws.microprofile.reactive.messaging_git_fat_tck/fat/src/com/ibm/ws/microprofile/faulttolerance/tck/ReactiveStreamsMessageingGitTckLauncher.java [hunk 7]
```diff
@@ -0,0 +1,191 @@
+/*******************************************************************************
+ * Copyright (c) 2018 IBM Corporation and others.
```
Comment (Azquelt) [inline dev/com.ibm.ws.microprofile.reactive.messaging_git_fat_tck/fat/src/com/ibm/ws/microprofile/faulttolerance/tck/ReactiveStreamsMessageingGitTckLauncher.java]: Update copyright year

File: dev/com.ibm.ws.microprofile.reactive.messaging_git_fat_tck/publish/servers/ReactiveStreamsTCKServer/server.xml [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+        <feature>componenttest-1.0</feature>
+        <feature>localConnector-1.0</feature>
+        <feature>cdi-2.0</feature>
+        <feature>mpReactiveStreams-1.0</feature>
+    </featureManager>
+
+    <include location="../fatTestPorts.xml" />
+
+    <!--Java2 security-->
+    <javaPermission className="java.security.AllPermission"  name="*" actions="*" />
```
Comment (Azquelt) [inline dev/com.ibm.ws.microprofile.reactive.messaging_git_fat_tck/publish/servers/ReactiveStreamsTCKServer/server.xml]: Remove this.

File: dev/com.ibm.ws.microprofile.reactive.messaging_git_fat_tck/fat/src/com/ibm/ws/microprofile/faulttolerance/tck/FATSuite.java [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+ * Copyright (c) 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.microprofile.faulttolerance.tck;
```
Comment (Azquelt) [inline dev/com.ibm.ws.microprofile.reactive.messaging_git_fat_tck/fat/src/com/ibm/ws/microprofile/faulttolerance/tck/FATSuite.java]: Wrong package.

File: dev/com.ibm.ws.microprofile.reactive.messaging_git_fat_tck/fat/src/com/ibm/ws/microprofile/faulttolerance/tck/ReactiveStreamsMessageingGitTckLauncher.java [hunk 10]
```diff
@@ -0,0 +1,156 @@
...
+ * Copyright (c) 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.microprofile.faulttolerance.tck;
```
Comment (Azquelt) [inline dev/com.ibm.ws.microprofile.reactive.messaging_git_fat_tck/fat/src/com/ibm/ws/microprofile/faulttolerance/tck/ReactiveStreamsMessageingGitTckLauncher.java]: Wrong package

File: dev/com.ibm.ws.microprofile.reactive.messaging_git_fat_tck/fat/src/com/ibm/ws/microprofile/faulttolerance/tck/ReactiveStreamsMessageingGitTckLauncher.java [hunk 11]
```diff
@@ -0,0 +1,156 @@
...
+import componenttest.annotation.Server;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.custom.junit.runner.Mode;
+import componenttest.custom.junit.runner.Mode.TestMode;
+import componenttest.custom.junit.runner.TestModeFilter;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.utils.MvnUtils;
+
+/**
+ * This is a test class that runs the whole Fault Tolerance TCK. The TCK results
```
Comment (Azquelt) [inline dev/com.ibm.ws.microprofile.reactive.messaging_git_fat_tck/fat/src/com/ibm/ws/microprofile/faulttolerance/tck/ReactiveStreamsMessageingGitTckLauncher.java]: "Fault Tolerance TCK"

File: dev/com.ibm.ws.microprofile.reactive.messaging_git_fat_tck/bnd.bnd [hunk 12]
```diff
@@ -0,0 +1,37 @@
...
```
Comment (Azquelt) [inline dev/com.ibm.ws.microprofile.reactive.messaging_git_fat_tck/bnd.bnd]: This list is still wrong.

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
#build

**Pull request merged**